cli/src/commands/env.ts

```diff
@@ -100,11 +118,15 @@ export default class Env extends Command {
     }
   }
 
-  protected async list(appId: string) {
-    const { variables } = await this.lycanClient.getEnv(appId, null);
-    // Output in sorted order (of variable names).
-    const sorted = variables.map(vvToString).sort();
-    this.log(sorted.join('\n'));
+  protected async list(appId: string, includeValues: boolean) {
+    try {
```

Comment:
No need for `try-catch` - I think errors are handled at a different level.

Reply:
Yuck, that's debugging code.  Squashing it out at the source (and removing this stain on code history).